backend/src/server/LogError.test.ts

```diff
@@ -0,0 +1,26 @@
+import { logger } from '@test/testSetup'
+
+import LogError from './LogError'
+
+describe('LogError', () => {
+  it('logs an Error when created', () => {
+    /* eslint-disable-next-line no-new */
+    new LogError('new LogError')
+    expect(logger.error).toBeCalledWith('new LogError')
+  })
+
+  it('logs an Error including additional data when created', () => {
+    /* eslint-disable-next-line no-new */
```

Comment:
Set this globally => one line less :)

Reply:
local scope is always safer and better than global scope.